src/app.py

```diff
@@ -19,6 +19,25 @@
 
 add_openapi_route(prefix_router)
 
+@prefix_router.post("/documents")
+def add_document(pdf: UploadFile, 
+        title: str = Form(), bucket: str = Form(), doi: Optional[str] = Form(None), 
+        x_api_key: str = Header()) -> Article:
+    """ Upload a new document to the document store """
+    if not (title and pdf.file and bucket):
```

Comment:
It's not clear to me how `title: str = Form(), bucket: str = Form()` works with this bool check. If a user does not provide a title or bucket, will this check work?

```
>>> bool(Form())
True
```

Reply:
FastAPI does a bunch of mangling of input data based on type annotations, so it automatically extracts the `str` value of title and bucket rather the raw `Form()` object. It looks from their documentation that the annotations I used are deprecated and a different from is preferred now: (`var: Annotated[str, Form()]` rather than `var: str = Form()`)